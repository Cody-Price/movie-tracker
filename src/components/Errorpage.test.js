import React from 'react'
import Errorpage from './ErrorPage'
import { shallow } from 'enzyme'


describe('Errorpage', () => {
	it('should match the snapshot', () => {
		const wrapper = shallow(<Errorpage />)
		expect(wrapper).toMatchSnapshot()
	})
})