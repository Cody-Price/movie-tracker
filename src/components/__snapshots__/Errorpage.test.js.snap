// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errorpage should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ErrorPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "The page you requested cannot be found. Please check your address and try again.",
      "className": "no-movies movie-container",
    },
    "ref": null,
    "rendered": "The page you requested cannot be found. Please check your address and try again.",
    "type": "h1",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "The page you requested cannot be found. Please check your address and try again.",
        "className": "no-movies movie-container",
      },
      "ref": null,
      "rendered": "The page you requested cannot be found. Please check your address and try again.",
      "type": "h1",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
